source/layouts/layout.slim

```diff
@@ -57,3 +57,9 @@ html.Theme itemscope='' itemtype='http://schema.org/WebPage' lang=data.site.lang
       try{Typekit.load();}catch(e){}
     = javascript_include_tag 'all'
     = yield_content :javascripts
+    - unless development?
```

Comment:
It is an actual thing:
https://github.com/middleman/middleman/blob/v3.0.6/middleman-core/lib/middleman-core/application.rb#L200-206

On Fri, Dec 11, 2015 at 4:57 PM Miguel Palhas <notifications@github.com> wrote:
In source/layouts/layout.slim :

> @@ -57,3 +57,9 @@ html.Theme itemscope='' itemtype='http://schema.org/WebPage' lang=data.site.lang
>        try{Typekit.load();}catch(e){}
>      = javascript_include_tag 'all'
>      = yield_content :javascripts
> +    - unless development?


did you test this? I don't think development? is really a thing. @zamith wrote that as pseudocode I think

We've been using if build? so far
example: https://github.com/subvisual/ subvisual.co/blob/ b4d53f1221e367c9489299ed354903
33234efe12/source/javascripts/ _segment.js.erb

—
Reply to this email directly or view it on GitHub .